tcms/templates/plan/get_cases.html

```diff
@@ -215,27 +186,3 @@
 	</span>
 	<span class="js-nomore-msg" style="display:none">All cases have been loaded.</span>
 </div>
-{% verbatim %}
-	<script id="batch_tag_form_template" type="text/x-handlebars-template">
-		<form id="id_batch_tag_form">
-			<div class="dia_title" style=" margin:30px 20px;">Please type tag name:</div>
-			<div class="dia_content" style=" margin:30px 20px;">
-				<input type="text" id="add_tag_plan" name="tags" style="width:300px; height:25px; border:solid  1px #ccc"/>
-				<div id="id_batch_add_tags_autocomplete"></div>
```

Comment:
Remove styles for id_batch_add_tags_autocomplete from CSS.

Reply:
Done